docker-compose.yaml

```diff
@@ -3,6 +3,25 @@ version: '3.8'
 name: manvsim
 
 services:
+
+  postgres:
+    image: postgres:latest
+    ports:
+      - 5432:5432
```

Comment:
Its recommended to use quotes here. Also i think i would like it better if the postgres service was below app, server and web

Reply:
done